package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.6.2",
+  "version": "0.6.3",
```

Comment:
i believe `main` is now at `6.3` so this might require another bump up

Reply:
👍